CHANGELOG.rst

```diff
@@ -22,6 +22,7 @@ Bugfixes
 - zabbix_action - will no longer wipe `esc_step_to` and `esc_step_from` (https://github.com/ansible-collections/community.zabbix/issues/692)
 - zabbix_agent - added support for zabbix-agent on Ubuntu 22.04 (https://github.com/ansible-collections/community.zabbix/pull/681)
 - zabbix_agent - now properly creates webroot for issuing LE certificates (https://github.com/ansible-collections/community.zabbix/pull/677, https://github.com/ansible-collections/community.zabbix/pull/682)
+- zabbix-agent - will no longer wipe tls_connect en tls_accept settings when not specified on update (https://github.com/ansible-collections/community.zabbix/pull/753)
```

Comment:
Changelog shouldn't go directly into the main CHANGELOG file. Please add fragment into /changelogs/fragments directory. See existing files as an examples. This should be categorized as minor_changes.
```suggestion
---
minor_changes:
  - zabbix_host - will no longer wipe tls_connect en tls_accept settings when not specified on update
```
Above should go into /changelogs/fragments/753-host-tls-settings.yml for example.